### ISSUE TITLE
Wither storm picking up

### ISSUE THREAD

**Issue by KrukAdrian** [nonamecrackers2/crackers-wither-storm-mod#424]
Make it so you can actually move in the wither storm and while moving the eye would move slowly so you could escape

**Comment by KrukAdrian** [comment]
wait i should have made this a bug

**Comment by nonamecrackers2** [comment]
Could you potentially rephrase this? Confused on what you're suggesting.

**Comment by KrukAdrian** [comment]
What i meant by that is when it picks you up with its tractor beam you barely can move and when you are still moving the tractor beam follows you, make it so you can move in the tractor beam and that it slowly moves so you can escape it

**Comment by DiamondWalker** [comment]
I agree with this. I've died in some annoying situations where escaping should've been really easy. One time it pulled me through a one block gap in a ravine lol.

**Comment by BluePineappleGuy** [comment]
moving around should also have the swimming animation but a bit slower
